h/templates/activity/search.html.jinja2

```diff
@@ -82,6 +82,14 @@
 
   {{ panel('navbar') }}
 
+  {% for url in asset_urls("search_js") %}
+    <script src="{{ url }}"></script>
```

Comment:
Unfortunately this is still going to block HTML parsing during download. If we add the `defer` attribute here and on the script tags which depend on it (in `base.html.jinja2`) this will avoid the problem since deferred scripts execute after HTML parsing and in-order.

Perhaps it would be best to revisit this after this PR is merged though.